mopidy/mpd/actor.py

```diff
@@ -60,7 +62,15 @@ def _setup_server(self, config, core):
                 'MPD server startup failed: %s' %
                 encoding.locale_decode(error))
 
-        logger.info('MPD server running at [%s]:%s', self.hostname, self.port)
+        if server.server_socket.type == socket.AF_UNIX:
+            logger.info('MPD server running at %s', self.hostname)
```

Comment:
Maybe we could have a helper method to format the string for us since it occurs in a few places?